Reorg and Move Cryptography Page

filippoweb3 opened the pull request:
See linked issue

Commit:
Moved content

from keys to cryptography

Commit:
Deleted keys page

also from sidebar, added cryptography page to Advanced

Commit:
Moved xcm alphabetically in sidebar

DrW3RK commented:
Added comments to the linked issue https://github.com/w3f/polkadot-wiki/issues/4105

Commit:
Added redirect for deleted keys page

filippoweb3 commented:
@DrW3RK thank for the suggestion. I think that the redirect option is better as it keeps the wiki structure clean allowing us to delete old pages or pages where we moved content. Let me know your opinion about this. I added the redirect for the learn-keys page that I deleted. Let me know if this is ok, if yes I will then do the same for the [other PR](https://github.com/w3f/polkadot-wiki/pull/4108).

Commit:
Added specific section to redirect

Commit:
removed section in redirect
